CHANGES.rst

```diff
@@ -10,11 +10,19 @@ Changes
 
 Changes:
 --------
-- No change.
+- | Modify problematic output location and execution methodology of ``file2string_array`` process so it does what
```

Comment:
Does it breaks any existing workflows in the wild? It seems that you fix a broken function, but I want to know if there is any back-compatibility issues.

Reply:
I believe the workflows are all broken for a long time. 
All tests that defined a workflow using `file2string_array` are decorated with `@pytest.mark.xfail`
https://github.com/crim-ca/weaver/blob/master/tests/functional/test_ems_end2end.py#L631-L658

When I look at the input/output of the steps, they don't make sense 
`file2string_array` (before the change) already indicated that it produces a single JSON file as output, but following `ColibriFlyingpigeon_SubsetBbox` takes a single NetCDF as input.
It's like there is an intermediate step missing, where one index should be specified to extract the NetCDF link from the generated JSON array by `file2string_array`.